$NetBSD: patch-Makefile,v 1.3 2012/08/25 13:01:42 christos Exp $

o Automatically get and use the iana files.
o Disable the install target.
o Update for the new filenames at IANA's website

--- Makefile	2012-08-25 13:45:59.000000000 +0300
+++ Makefile	2012-08-25 13:10:09.000000000 +0300
@@ -17,7 +17,7 @@
 .PHONY: all files get test test-services test-protocols install clean \
 	protocol-numbers.iana port-numbers.iana dist
 
-all: files
+all: get files
 files: protocols services
 
 get: protocol-numbers.iana port-numbers.iana
@@ -30,10 +30,10 @@
 test-protocols: protocols test-lib.gawk test-protocols.gawk
 	$(AWK) -f test-lib.gawk -f test-protocols.gawk <protocols
 
-install: files
-	install -d $(DESTDIR)$(PREFIX)$(ETC_DIR)
-	install -m 644 protocols $(DESTDIR)$(PREFIX)$(ETC_DIR)
-	install -m 644 services $(DESTDIR)$(PREFIX)$(ETC_DIR)
+#install: files
+#	install -d $(DESTDIR)$(PREFIX)$(ETC_DIR)
+#	install -m 644 protocols $(DESTDIR)$(PREFIX)$(ETC_DIR)
+#	install -m 644 services $(DESTDIR)$(PREFIX)$(ETC_DIR)
 
 clean:
 	rm -vf \
@@ -42,26 +42,26 @@
 	    protocol-numbers.iana port-numbers.iana
 
 protocol-numbers.iana:
-	$(AWK) -f get.gawk -v file=protocol-numbers >protocol-numbers.iana
+	$(AWK) -f get.gawk -v url=protocol-numbers file=protocol-numbers >protocol-numbers.iana
 	rm -f protocol-numbers
 
 port-numbers.iana:
-	$(AWK) -f get.gawk -v file=port-numbers >port-numbers.iana
+	$(AWK) -f get.gawk -v url=service-names-port-numbers file=port-numbers >port-numbers.iana
 	rm -f port-numbers
 
 protocol-numbers:
-ifeq (protocol-numbers.iana, $(wildcard protocol-numbers.iana))
+#ifeq (protocol-numbers.iana, $(wildcard protocol-numbers.iana))
 	ln -f -s protocol-numbers.iana protocol-numbers
-else
-	ln -f -s protocol-numbers.dist protocol-numbers
-endif
+#else
+#	ln -f -s protocol-numbers.dist protocol-numbers
+#endif
 
 port-numbers:
-ifeq (port-numbers.iana, $(wildcard port-numbers.iana))
+#ifeq (port-numbers.iana, $(wildcard port-numbers.iana))
 	ln -f -s port-numbers.iana port-numbers
-else
-	ln -f -s port-numbers.dist port-numbers
-endif
+#else
+#	ln -f -s port-numbers.dist port-numbers
+#endif
 
 protocols: protocol-numbers protocols.gawk
 	$(AWK) --re-interval -f protocols.gawk -v strip=$(STRIP) \
